react/features/toolbox/defaultToolbarButtons.js

```diff
@@ -180,6 +152,23 @@ export default {
     },
 
     /**
+     * The descriptor of the dial out toolbar button.
+     */
+    dialOut: {
+        classNames: [ 'button', 'icon-telephone' ],
+        enabled: true,
+
+        // Will be displayed once the SIP calls functionality is detected.
+        hidden: true,
+        id: 'toolbar_button_dial_out',
+        onClick() {
+            JitsiMeetJS.analytics.sendEvent('toolbar.sip.clicked');
+            APP.store.dispatch(openDialOutDialog());
```

Comment:
I thought we are supposed to use APP.store global only for the not yet reactified code. Isn't that true?